app/src/main/res/layout/gpodnetauth_credentials.xml

```diff
@@ -33,7 +33,11 @@
         android:layout_margin="8dp"
         android:focusable="true"
         android:focusableInTouchMode="true"
-        android:cursorVisible="true"/>
+        android:cursorVisible="true"
+        android:maxLines="1"
```

Comment:
You might want to take a look at https://developer.android.com/reference/android/widget/TextView.html#attr_android:inputType and https://developer.android.com/reference/android/text/InputType.html
if inputType is not set, it apparently defaults to textMultiLine for EditText